client/views/root/canada_post.js

```diff
@@ -0,0 +1,3 @@
+import RootView from '../generic-settings-form';
```

Comment:
> so right now we would have shipping, self-help, and shipping-label. Is that ok?

Yes, I think something like that would work. Is shipping-label more specific than needed? I.e. if we wanted to add another Button -> Dialog thing down the road to Edit Order.